docs/development.rst

```diff
@@ -23,6 +23,17 @@ Generate minified JS and CSS files via ``Gulp``:
 Run the test suite with ``tox``. More information about testing is
 available at :doc:`Testing page <testing>`.
 
+Local testing
+-------------
+
+This extension works when is hosted on Read the Docs,
+to make it work locally a custom ``READTHEDOCS_DATA`` js variable is provided to send the search requests to
+https://readthedocs-sphinx-search.readthedocs.io/_/api/v2/search/.
+Due to our CORS settings your browser will block the requests,
+so you'll need to install a plugin like `CORS everywhere`_ to make it work.
```

Comment:
Can we use jsonp in this case to work around this issue?

Reply:
Just tested it, our backend needs to respond with `application/jsonp` as well, thing we don't support in our api.

Reply:
Sure, but most of our API endpoints support jsonp, I don't think it would be hard to support here, and much less hacky :)

Reply:
Isn't it just adding this: 

https://github.com/readthedocs/readthedocs.org/blob/8e78d680d02aeba12644796b979ef62459f64932/readthedocs/api/v2/views/footer_views.py#L91

Reply:
yeah, we would need to add that, but I'm not really in favor of solving our cors problems with jsonp, maybe we should just add the cors header to the api to allow all domains instead.

Reply:
That is probably a better solution 👍

Reply:
We have CORS header now! I have removed this section now.